scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js

```diff
@@ -47,7 +47,10 @@ export function ItemAssociationDirective(superdesk, renditions, config, authorin
             function getItem(event, dataType) {
                 let item = angular.fromJson(event.originalEvent.dataTransfer.getData(dataType));
 
-                if (item._type !== 'externalsource') {
+                if (item._type === 'archived') {
+                    return api.find('archived', item._id)
+                        .then((result) => result);
+                } else if (item._type !== 'externalsource') {
                     return api.find('archive', item._id)
```

Comment:
maybe it should just do `if (item._type !== 'externalsource') { return api.find(item._type, item._id); }`